reference/MarkdownViewer/MarkdownViewer.upp

```diff
@@ -0,0 +1,15 @@
+description "A simple markdown viewer with image displaying capability.\377";
+
+uses
+	CtrlLib,
+	Painter,
+	plugin/jpg,
+	plugin/md;
+
+file
+	main.cpp;
+
+mainconfig
+	"" = "GUI",
+	"" = "GUI .NOGTK";
```

Comment:
Not sure if we need this flag. One GUI flag should be enough here.

Reply:
Done.